zigbee-console-common/src/main/java/org/bubblecloud/zigbee/ZigBeeConsole.java

```diff
@@ -499,7 +515,15 @@ public boolean process(final ZigBeeApi zigbeeApi, final String[] args) {
             print("Device Version   : " + device.getDeviceVersion());
             print("Implementation   : " + device.getClass().getName());
             print("Input Clusters   : ");
-            for (int c : device.getInputClusters()) {
+            showClusters(device, true);
```

Comment:
If you're going to change this, wouldn't a better implementation be to pass in the cluster list (ie either input or output clusters) rather than a true/false?